aws/resource_aws_organizations_organization.go

```diff
@@ -137,6 +137,7 @@ func resourceAwsOrganizationsOrganization() *schema.Resource {
 					Type: schema.TypeString,
 					ValidateFunc: validation.StringInSlice([]string{
 						organizations.PolicyTypeServiceControlPolicy,
+						organizations.PolicyTypeTagPolicy,
```

Comment:
This new value should be documented in `website/docs/r/organizations_organization.html.markdown`

Reply:
The policy type is documented in the linked aws documentation, but I've added the value as an example.